ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dot1x.j2

```diff
@@ -79,5 +99,27 @@ dot1x
    radius av-pair framed-mtu {{ dot1x.radius_av_pair.framed_mtu }}
 {%             endif %}
 {%         endif %}
+{%         if dot1x.supplicant.disconnect_cached_results_timeout is arista.avd.defined %}
+   supplicant disconnect cached-results timeout {{ dot1x.supplicant.disconnect_cached_results_timeout }} seconds
+{%         endif %}
+{%         if dot1x.captive_portal.enabled is arista.avd.defined(true) %}
+{%             set captive_portal_cli = "captive-portal" %}
+{%             if dot1x.captive_portal.url is arista.avd.defined %}
+{%                 set captive_portal_cli = captive_portal_cli ~ " url " ~ dot1x.captive_portal.url %}
+{%             endif %}
+{%             if dot1x.captive_portal.ssl_profile is arista.avd.defined %}
+{%                 set captive_portal_cli = captive_portal_cli ~ " ssl profile " ~ dot1x.captive_portal.ssl_profile %}
+{%             endif %}
+   {{ captive_portal_cli }}
+{%             if dot1x.captive_portal.access_list_ipv4 is arista.avd.defined %}
+   captive-portal access-list ipv4 {{ dot1x.captive_portal.access_list_ipv4 }}
+{%             endif %}
+{%         endif %}
+{%         if dot1x.captive_portal.start_limit_infinite is arista.avd.defined(true) %}
+   captive-portal start limit infinite
+{%         endif %}
```

Comment:
I think you want this last line to be inside the if for `enabled`

Reply:
yes, thanks!

Fixed